### PR TITLE
fix: removed checkboxes and circles from pages

### PR DESCRIPTION
**Pull request by ykethan**
#### Description of changes:

Removes checkboxes: https://docs.amplify.aws/react/build-a-backend/data/connect-to-existing-data-sources/connect-postgres-mysql-database/
image

removes circle for consistency: https://docs.amplify.aws/react/start/migrate-to-gen2/#functions
image

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by josefaidt — APPROVED** [review]

**Review by hbuchel — COMMENTED** [review]
I [opened a PR](https://github.com/aws-amplify/docs/pull/7784) to fix the failing accessibility issue since it's not content related; once that is merged in you can pull latest and it should unblock this PR.

**Review by hbuchel — APPROVED** [review]

**Pull request merged**
